src/taskpane/server.js

```diff
@@ -2897,7 +3484,32 @@ function createEntryFromRow(row, model, fieldTypeEnums, artifactIsHierarchical,
             customType = "DateTimeValue";
           }
           break;
-
+        case fieldTypeEnums.customDate:
+          if (row[index]) {
+            if (IS_GOOGLE) {
+              value = row[index];
+            } else {
+              // for Excel, dates are returned as days since 1900 - so we need to adjust this for JS date formats
+              const DAYS_BETWEEN_1900_1970 = 25567 + 2;

```

Comment:
suggest you put this const at the top of the file and then you do not need to declare it every time and in both excel and google